src/xs_driver.cpp

```diff
@@ -714,11 +714,11 @@ bool InterbotixDriverXS::get_joint_states(
     if (positions) {
       if (is_motor_gripper(joint_name)) {
         positions->push_back(
-          robot_positions.at(get_js_index(joint_name)) - \
-          gripper_map[joint_name].calibration_offset);
+          robot_positions.at(get_js_index(joint_name)));
+          // gripper_map[joint_name].calibration_offset);
         double pos = convert_angular_position_to_linear(
           joint_name,
-          positions->at(get_js_index(joint_name)) + gripper_map[joint_name].calibration_offset);
+          positions->at(get_js_index(joint_name)));
```

Comment:
Removing calibration offset here?

Reply:
Yes this was one bug causing the problem of offset in gripper. The position of the was already updated with the offset.
Also, now that we are initializing the operating modes after gripper calibration. We do not actually need the offset anymore.